dev/io.openliberty.persistence.3.2.thirdparty/resources/org/eclipse/persistence/internal/localization/i18n/LoggingLocalizationResource.nlsprops

```diff
@@ -193,6 +201,8 @@ error_in_remove=Error in remove.
 
 error_loading_xml_file=Exception while loading ORM xml file: {0}: {1}
 
+error_queryTimeoutParse=Could not parse jakarta.persistence.query.timeout property value {0}: {1}
```

Comment:
Change to:

Cannot parse the {0} jakarta.persistence.query.timeout property value: {1}